components/x-teaser/src/Status.jsx

```diff
@@ -2,14 +2,17 @@ import { h } from '@financial-times/x-engine'
 import TimeStamp from './TimeStamp'
 import RelativeTime from './RelativeTime'
 import LiveBlogStatus from './LiveBlogStatus'
+import AlwaysShowTimestamp from './AlwaysShowTimestamp'
 
 export default (props) => {
 	if (props.status) {
 		return <LiveBlogStatus {...props} />
 	}
 
 	if (props.publishedDate) {
-		if (props.useRelativeTime) {
+		if (props.alwaysDisplayTimestamp) {
```

Comment:
**Thoughts**: Do you have any thoughts on the choice of name? I didn't want to make it too big, but then again difficult to portray what it is doing - 'Relative time for the same day, non-relative for any other'.

Other thought I have is to, plug in the name of the work that it is tied with e.g. how LiveBlogStatus is done. In this case it would be something along the line of 'NewsfeedTimestamp'